src/Routers/CloudCodeRouter.js

```diff
@@ -53,6 +55,24 @@ export class CloudCodeRouter extends PromiseRouter {
       middleware.promiseEnforceMasterKeyAccess,
       CloudCodeRouter.deleteJob
     );
+    this.route(
+      'GET',
+      '/releases/latest',
+      middleware.promiseEnforceMasterKeyAccess,
+      CloudCodeRouter.getCloudCode
+    );
+    this.route(
+      'GET',
+      '/scripts/*',
+      middleware.promiseEnforceMasterKeyAccess,
+      CloudCodeRouter.getCloudFile
+    );
+    this.route(
+      'POST',
+      '/scripts/*',
+      middleware.promiseEnforceMasterKeyAccess,
+      CloudCodeRouter.saveCloudFile
+    );
   }
```

Comment:
Even if master key is enforced may be we should have a Parse Server option to disable this feature 😉